Tasks/DockerInstallerV0/task.loc.json

```diff
@@ -0,0 +1,59 @@
+{
+  "id": "8e038650-f7bf-11e8-8a6c-8fff434b4eff",
+  "name": "DockerInstaller",
+  "friendlyName": "ms-resource:loc.friendlyName",
+  "description": "ms-resource:loc.description",
+  "helpMarkDown": "ms-resource:loc.helpMarkDown",
+  "category": "Tool",
+  "visibility": [
+    "Build",
+    "Release"
+  ],
+  "author": "Microsoft Corporation",
+  "version": {
+    "Major": 0,
+    "Minor": 1,
+    "Patch": 0
+  },
+  "demands": [],
+  "satisfies": [
+    "Docker"
+  ],
+  "minimumAgentVersion": "2.142.1",
+  "inputs": [
+    {
+      "name": "dockerVersion",
+      "label": "ms-resource:loc.input.label.dockerVersion",
+      "type": "string",
+      "required": true,
+      "helpMarkDown": "ms-resource:loc.input.help.dockerVersion",
+      "defaultValue": "17.09.0-ce"
```

Comment:
Is this latest?

Reply:
Yes. This is the latest common version across all platforms (windows is at 17.09.0-ce. Mac and linux are at 18.09.0

Reply:
Is this version available across all platforms?